packages/aws-cdk-lib/custom-resources/test/custom-resource-config/custom-resource-config.test.ts

```diff
@@ -200,3 +200,54 @@ test('addLogRetentionLifetime modifies the retention period of the custom resour
     RetentionInDays: customResourceLogRetention,
   });
 });
+
+describe('when custom resource logGroup is default set to Retain', () => {
```

Comment:
"default" always means a value that we set if a property / flag / option is undefined. Using the language "when ...default is set to X" suggests that this test is controlling the default of the log group's retention, which it isn't; it's controlling that property.